GeosCore/fullchem_mod.F90

```diff
@@ -583,9 +600,8 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
 
        ! Check if the current grid cell in this loop should have its
        ! full chemical state printed (concentrations, rates, constants)
-       ! for use with the KPP Standalone
-       ! (psturm, 03/22/24)
-       CALL Check_ActiveCell( I, J, L, State_Grid )
+       ! for use with the KPP Standalone (psturm, 03/22/24)
+       CALL KppSa_Check_ActiveCell( I, J, L )
```

Comment:
Good call, State_Grid was only used for debugging. If users want to see how far their sampled grid cell midpoints are from the assigned locations, they can compare output files to the yaml file coordinates.